CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Contract code
+/src @smartcontractkit/research @smartcontractkit/security
```

Comment:
@connorwstein seems we would need to ask sec to give these teams write access as a whole first so removing for now and sec tickets takes days to get done. i will add it after